sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java

```diff
@@ -298,11 +302,14 @@ private Flux<CosmosBulkOperationResponse<TContext>> executeCore() {
 
                 return this.inputOperations
                     .publishOn(CosmosSchedulers.BULK_EXECUTOR_BOUNDED_ELASTIC)
-                    .onErrorContinue((throwable, o) ->
-                        logger.error("Skipping an error operation while processing {}. Cause: {}, Context: {}",
-                            o,
+                    .onErrorMap(throwable -> {
```

Comment:
This is just a transformation operator, so it won't change what caused the error. Shouldn't we use `onErrorResume()` to skip the error causing input operation, and then resume with rest of the flow?

Reply:
Why would we want to skip the error - that was the behavior onErrorcContinue did - but if you pass a Flux into bulkExecution and there are failures you would want to know about, wouldn't you? And if you would want to suppress it that is something you could do in the Flux. I don't think this silently swallowing errors should happen within the Spark connector?